### PR TITLE
update to 1.19.3

### PR DESCRIPTION
**Pull request by moehreag**
- cleanup of the mixin (yes, it only needs to be that simple! and now it works on more versions as well!)
- remove mention of a "CreeperEntityRendererMixin" which was maybe forgotten to be added?

### CODE REVIEW

File: gradle.properties [hunk 1]
```diff
@@ -1,6 +1,6 @@
 org.gradle.jvmargs=-Xmx2G
 org.gradle.parallel=true
 org.gradle.caching=true
-minecraft=1.19.2
```
Comment (shateq) [inline gradle.properties:4]: Usual

File: src/main/java/shateq/tnttime/mixin/TntEntityRendererMixin.java [hunk 2]
```diff
@@ -23,33 +19,10 @@ protected TntEntityRendererMixin(EntityRendererFactory.Context ctx) {
...
-            int backgroundColor = (int) (backgroundOpacity * 255.0F) << 24;
-
-            final TextRenderer textRenderer = this.getTextRenderer();
-            float x = (float) (-textRenderer.getWidth(text) / 2);
-            float y = -4;
-
-            textRenderer.draw(text, x, y, 7, true, matrix4f, vertexConsumers, false, backgroundColor, light);
-            matrices.pop();
-        }
+    @Inject(method = "render(Lnet/minecraft/entity/TntEntity;FFLnet/minecraft/client/util/math/MatrixStack;Lnet/minecraft/client/render/VertexConsumerProvider;I)V", at = @At(value = "INVOKE", target = "Lnet/minecraft/client/render/entity/EntityRenderer;render(Lnet/minecraft/entity/Entity;FFLnet/minecraft/client/util/math/MatrixStack;Lnet/minecraft/client/render/VertexConsumerProvider;I)V"))
```
Comment (shateq) [inline src/main/java/shateq/tnttime/mixin/TntEntityRendererMixin.java:22]: Interesting, but I'd like to test it a bit (IT'S GOOD)

File: src/main/resources/tnttime.mixins.json [hunk 3]
```diff
@@ -6,7 +6,6 @@
   "compatibilityLevel": "JAVA_17",
   "mixins": [],
   "client": [
-    "CreeperEntityRendererMixin",
```
Comment (shateq) [inline src/main/resources/tnttime.mixins.json:9]: CreeperEntityRenderer is nearly ready, I guess I will have to squash it to another branch for the time :)

**Review by shateq — APPROVED** [review]
nice, give me a week

**Comment by shateq** [comment]
@moehreag your contributions are cool fireworks effective, you're great, don't change it!

**Comment by moehreag** [comment]
> @moehreag your contributions are cool fireworks effective, you're great, don't change it!

Thanks!

**Pull request merged**